src/rtpmidid.cpp

```diff
@@ -302,7 +302,8 @@ void rtpmidid_t::connect_client(const std::string &name, int aseq_port) {
         });
     peer_info->use_count++;
 
-    peer_info->peer->connect_to(address.address, address.port);
+    auto local_port = 50010;
```

Comment:
If we can make this a command line option (0 meaning no fixed port, every client their own). I think we can merge this change as it does not change that much, just makes it more tolerant.